docs/misc/gametest.md

```diff
@@ -247,6 +247,19 @@ property 'forge.enabledGameTestNamespaces', 'modid1,modid2,modid3'
 
 The Game Test Server is a special configuration which runs a build server. The build server returns an exit code of the number of required, failed Game Tests. All failed tests, whether required or optional, are logged. This server can be run using `gradlew runGameTestServer`.
 
+!!! important
+    Due to a quirk in how Gradle works, by default, if a task forces a system exit before it has finished executing, even in a zero code exit, the task will be marked as the failure. ForgeGradle sets by default a force exit on run tasks such that any subprojects are not executed in sequence. However, as such, the Game Test Server will always fail.
```

Comment:
Note that the actual behavior is that the task action runs `System.exit` which _kills the Gradle daemon_, causing the Gradle runner to report a build failure.

Reply:
I've changed the wording on this to hopefully reflect that point.